lib/components/Icons/MetamaskFox.tsx

```diff
@@ -1,106 +1,105 @@
 import React from "react";
 
-export function MetaMaskFox() {
-  return (
-    <svg
-      xmlns="http://www.w3.org/2000/svg"
-      x="0"
-      y="0"
-      enableBackground="new 0 0 318.6 318.6"
-      version="1.1"
-      viewBox="0 0 318.6 318.6"
-      xmlSpace="preserve"
-    >
-      <path d="M274.1 35.5L174.6 109.4 193 65.8z" className="st0"></path>
-      <path d="M44.4 35.5L143.1 110.1 125.6 65.8z" className="st1"></path>
+export const MetaMaskFox = () => (
```

Comment:
thanks for cleaning these up!
Would you mind, since you are in here, aligning all the SVGs to use this pattern:

```tsx
interface Props {
  className: string;
}

const MySVG = (props: Props) => {
  return <svg className={props.className} />
}
```

Namely:
- include a classname prop for each
- [dont destructure props](https://github.com/KlimaDAO/klimadao/wiki#do-not-destructure-props)

Reply:
Easy, I'll spend a bit of time polishing these SVG files to follow our conventions.
Do we have a preference to be explicit with the `return` keyboard?

Reply:
neat thanks! I have no preference for `return` keyword